flink-table/flink-table-common/src/main/java/org/apache/flink/table/expressions/UnresolvedCallExpression.java

```diff
@@ -45,13 +45,19 @@ public UnresolvedCallExpression(FunctionDefinition functionDefinition, List<Expr
 		this.args = Collections.unmodifiableList(new ArrayList<>(Preconditions.checkNotNull(args)));
 	}
 
+	public FunctionDefinition getFunctionDefinition() {
+		return functionDefinition;
+	}
+
 	@Override
-	public List<Expression> getChildren() {
-		return this.args;
+	public String asSummaryString() {
+		final List<String> argList = args.stream().map(Object::toString).collect(Collectors.toList());
```

Comment:
nit: you could do `collect(Collectors.joining("(", ", ", ")");`

Feel free to ignore though.

Reply:
or even better: also call `asSummaryString` there. thanks for the hint :)